fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -651,6 +651,8 @@ private void setDateValues(DateParmVal p, Date date) {
             p.setValueDateStart(DateTimeHandler.generateTimestamp(inst));
             inst = DateTimeHandler.generateUpperBound(date.getValue());
             p.setValueDateEnd(DateTimeHandler.generateTimestamp(inst));
+            
+            System.out.println(p);
```

Comment:
note to self:  make sure we remove this before merging

Reply:
should be removed